services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/android/attestation/AttestationStatement.java

```diff
@@ -147,4 +149,31 @@ public boolean isEvaluationTypeEqualTo(EvaluationType evType) {
     }
     return false;
   }
-}
\ No newline at end of file
+
```

Comment:
Question: Wouldn't be better to have all the equals and hash methods developed in the same way?  see {ExposureWindow.java}

Reply:
the one existing is generated code, just like this one, the advantage of this one is that sonar doesn't raise a critical / blocker for it being too complex. it would arguably be better to change all the old ones, but i chose to place a comment on the method to prevent sonar.